src/pydocstyle/parser.py

```diff
@@ -102,7 +102,7 @@ def __str__(self):
 class Module(Definition):
     """A Python source code module."""
 
-    _fields = ('name', '_source', 'start', 'end', 'decorators', 'docstring',
+    _fields = ('name', '_source', 'start', 'end', 'decorators', 'docstring',  # type: ignore
```

Comment:
Why?

Reply:
Overriding the `_fields` member of the base class:
```
py37-tests runtests: commands[1] | mypy --config-file=tox.ini src/
src\pydocstyle\parser.py:105: error: Incompatible types in assignment (expression has type "Tuple[str, str, str, str, str, str, str, str, str, str, str, str]", base class "Definition" defined the type as "Tuple[str, str, str, str, str, str, str, str, str]")
```